### PR TITLE
Add example configs

### PR DESCRIPTION
**Pull request by stephen-lt-sl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,4 +38,35 @@ Generate your own access token and save it in the environment variable `TWITTER_
...
+`eventConfig.json` has the following format:
+```JSON5
+{ 
+  // An array of hashtags, where tweets that use these hashtags will be displayed by the twitter wall
+  "hashtags":["#bristech", "#bristech2016"],
+  // An array of twitter users, where tweets that mention these users will be displayed by the twitter wall
+  "mentions":["@bristech"],
+  // A twitter user, where all tweets from the user will be displayed by the twitter wall
+  "officialUser":"bristech",
+  // An array of users, where tweets from these users that use any of the above hashtags will be prioritized by the twitter wall
```
Comment (susanthomson) [inline README.md]: prioritized? :us:

Comment (stephen-lt-sl) [reply README.md]: Yeah, couldn't think of a better way to put it that fit within a comment

Comment (susanthomson) [reply README.md]: prioritiSed :gb:

**Review by susanthomson — APPROVED** [review]
:jack_o_lantern:

**Pull request merged**